src/Pickles/Pickles.Test/WhenParsingCommandLineArguments.cs

```diff
@@ -487,5 +487,19 @@ public void ThenSetsLanguageToEnglishByDefault()
 
             Check.That(configuration.Language).IsEqualTo("en");
         }
+
+        [Test]
+        public void ThenCanParseIgnoreTagSuccessfully()
+        {
+            var ignoreTag = "ignore-tag";
```

Comment:
I'm happy that you include tests on your own initiative. I'm also happy about the style.

One detail I'd like you to change: I prefer Roy Osherhove's unit testing style of making values explicit and not using parameters.

So instead of (for example) `Check.That(configuration.IgnoreTag).IsEqualTo(ignoreTag)` I prefer `Check.That(configuration.IgnoreTag).IsEqualTo("ignore-tag")`.

The reason is that the second style makes the test easier to read: I instantly know what value we are talking about, and I don't need to check the value of some variable.